server/session/event_log.go

```
@@ -218,12 +219,23 @@ func readEventsFile(path string) (parsedEvents, error) {
 }
 
 func walkEventsFile(path string, visit func(Event) error) (parsedEvents, error) {
+	return walkEventsFileFromOffset(path, 0, visit)
+}
+
+func walkEventsFileFromOffset(path string, offset int64, visit func(Event) error) (parsedEvents, error) {
 	fp, err := openRegularSessionFile(path, "events file")
 	if err != nil {
 		return parsedEvents{}, fmt.Errorf("open events file: %w", err)
 	}
 	defer fp.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `fp.Close` is not checked

(errcheck)